frontend/app/routes/protected/apply/$id/application-delegate.tsx

```diff
@@ -41,6 +42,9 @@ export async function loader({ context: { appContainer, session }, params, reque
   const t = await getFixedT(request, handle.i18nNamespaces);
   const meta = { title: t('gcweb:meta.title.template', { title: t('protected-apply:application-delegate.page-title') }) };
 
+  const idToken: IdToken = session.get('idToken');
+  appContainer.get(TYPES.domain.services.AuditService).createAudit('page-view.renew.apply.application-delegate', { userId: idToken.sub });
```

Comment:
'page-view.renew.apply.application-delegate'

`renew` should be changed to `apply` for all audits :)